src/jit/rangecheck.cpp

```diff
@@ -208,7 +208,11 @@ void RangeCheck::OptimizeRangeCheck(BasicBlock* block, GenTreePtr stmt, GenTreeP
 
     // If we are not looking at array bounds check, bail.
     GenTreePtr tree = treeParent->gtOp.gtOp1;
-    if (tree->gtOper != GT_ARR_BOUNDS_CHECK)
+    if (tree->gtOper != GT_ARR_BOUNDS_CHECK
```

Comment:
Perhaps something like `GenTree::IsBoundsCheck()` is warranted to handle the code that is conditional on `FEATURE_SIMD`?